src/cli/sequencerSteps.js

```diff
@@ -9,44 +9,31 @@ module.exports = function (program, sequencer, outputFilename) {
     if (program.basic)
       console.log('Basic mode is enabled, outputting only final image');
 
-    // Iterate through the steps and retrieve their inputs.
-    program.step.forEach(function(step) {
-      var options = Object.assign({}, sequencer.modulesInfo(step).inputs);
+
+    program.steps.forEach(function(step) {
+      var options = Object.assign({}, sequencer.modulesInfo(step.name).inputs);
 
       // If inputs exists, print to console.
       if (Object.keys(options).length) {
-        console.log('[' + step + ']: Inputs');
+        console.log('[' + step.name + ']: Inputs');
       }
 
       // If inputs exists, print them out with descriptions.
       Object.keys(options).forEach(function(input) {
         // The array below creates a variable number of spaces. This is done with (length + 1).
         // The extra 4 that makes it (length + 5) is to account for the []: characters
         console.log(
-          new Array(step.length + 5).join(' ') +
+          new Array(step.name.length + 5).join(' ') +
           input +
           ': ' +
           options[input].desc
         );
       });
 
-      if (program.config) {
```

Comment:
What was this and why was it removed?

Reply:
This was used to get input for options for different modules using `-c` flag, I removed this to support new format.

Reply:
Now we can't directly give input? Or is there a different flag for it now?

Reply:
Ah just need clarification on this - thanks!

Reply:
Before:
`./index.js -s "webgl-distort" -c '{"nw":"100,100","se":"40,40"}' -i examples/images/test.png`
Now:
`./index.js -s "webgl-distort{nw:100%2C100|se:40%2C40}" -i examples/images/test.png`

So since we used to give options seperately therefore we needed the `-c` flag but now the option values are given inside the same string(just like the url) so it isn't needed anymore.